README.md

```diff
@@ -1 +1,148 @@
-# budget-app
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>budget App</b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [budget App ](#budget-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+  - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [Acknowledgments](#acknowledgments)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# budget App <a name="about-project"></a>
+
+> Budget app is an app for managing transactions with categorized lists, sign-up and login pages, and navigation buttons for moving between pages. Users can add new categories or transactions.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby On Rails</a></li>
+  </ul>
+</details>
+
+<p align="right"><a href="#readme-top">back to top</a></p>
+
+<!-- Key Features -->
+
+## Key Features <a name="key-features"></a>
+
+- [x] Users can create an account and log in.
+- [x] Login users can create a new budget.
+- [x] Login users can add ingredients to a budget.
+- [x] Login users can delete a budget.
+
+<!-- GETTING STARTED -->
+
```

Comment:
- [ ] Kindly add the live demo section and add the deployed link on it since this project is deployed in order to keep your README professional.